packages/next/package.json

```
@@ -69,7 +69,7 @@
     "@trpc/client": "10.33.0",
     "@trpc/react-query": "10.33.0",
     "@trpc/server": "10.33.0",
-    "next": "13.4.7",
+    "next": "13.4.8",
```

```suggestion
    "next": "*",
```

Idk why but this is manypkg who bumps the peerDep when you bump the dependency in other place :/ real annoying.